test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -171,5 +195,38 @@ defmodule DotcomWeb.Live.TripPlannerTest do
 
       refute render_async(view) =~ "View All Options"
     end
+
+    test "'Depart At' buttons toggle which itinerary to show", %{
+      conn: conn,
+      params: params
+    } do
+      base_itinerary =
+        OtpFactory.build(:itinerary)
+        |> limit_route_types()
+
+      # Right now, the headsign (which is what we actually want to
+      # show) is not available from OTP client, but we're rendering
+      # the trip ID in its place. Once the headsign is available, we
+      # should update these updates and the assertions below to use
+      # the headsign instead of the trip ID.
+      stub_otp_results([
+        update_trip_details(base_itinerary, trip_id: "trip_id_1", start_time: ~T[09:26:00]),
+        update_trip_details(base_itinerary, trip_id: "trip_id_2", start_time: ~T[10:46:00])
```

Comment:
I went the way I did here because I prefer not to have logic in tests, since that makes it harder to see exactly what the test is testing. In this case, the logic would be the conversion from `~T[09:26:00]` into the string `"9:26AM"`. 

I do agree that having `~T[09:26:00]` and `"9:26AM"` be closer to each other in the test file would make them feel less magic. Lemme try a quick refactor, and then let's talk if we still don't love that.